hack/test-integration.sh

```diff
@@ -96,16 +101,33 @@ loop="${TIMES:-1}"
 # hack/test-integration.sh Template*
 # hack/test-integration.sh "(WatchBuilds|Template)"
 tests=( $(go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testexec}" | grep -E "${1-Test}") )
+
+if [[ -n "${JUNIT_REPORT:-}" ]]; then
```

Comment:
Move this up to line 32 for visual clarity -- should execute fine but is weird that it's after the `exectest()` declaration.